packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -382,24 +380,6 @@ class RenderingEngine implements IRenderingEngine {
     return viewports.filter(isStackViewport) as Array<IStackViewport>;
   }
```

Comment:
Not sure why

Reply:
The method was unused, and I don't see a need for methods for each type when we have the generic ones.